results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/SRNCorpusBitextMining.json

```diff
@@ -0,0 +1,22 @@
+{
+  "dataset_revision": "2903226ff89ca0b15221a75d32b6355248295119",
+  "mteb_dataset_name": "SRNCorpusBitextMining",
+  "mteb_version": "1.7.24",
+  "test": {
+    "evaluation_time": 2.89,
+    "nld_Latn-srn_Latn": {
+      "accuracy": 0.15234375,
+      "f1": 0.11054687499999999,
+      "main_score": 0.11054687499999999,
+      "precision": 0.09544270833333332,
+      "recall": 0.15234375
+    },
+    "srn_Latn-nld_Latn": {
```

Comment:
The accuracy is very low with this model 🤔

Reply:
Indeed, probably because Sranantongo is extremely low-resource. Performance for the other model is  better but still far from good.

Reply:
Curious how e5-base compares.

Reply:
Surprisingly (?), not better than e5-small:

```python
{
  "dataset_revision": "2903226ff89ca0b15221a75d32b6355248295119",
  "mteb_dataset_name": "SRNCorpusBitextMining",
  "mteb_version": "1.7.24",
  "test": {
    "evaluation_time": 6.98,
    "nld_Latn-srn_Latn": {
      "accuracy": 0.43359375,
      "f1": 0.3622147817460318,
      "main_score": 0.3622147817460318,
      "precision": 0.3375108506944444,
      "recall": 0.43359375
    },
    "srn_Latn-nld_Latn": {
      "accuracy": 0.30859375,
      "f1": 0.2417665234071484,
      "main_score": 0.2417665234071484,
      "precision": 0.22415235477476975,
      "recall": 0.30859375
    }
  }
```